content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+`dvc.yaml` files. In this case, you can run all pipelines with a single command:
+
+```dvc
+$ dvc exp run --all-pipelines
+```
+
+> Note that the order in which pipelines are executed is not guaranteed; Only
+> the internal order of stage execution is.
+
+### Interactive reproduction
+
+When you want to have more granular control over which stages are run, you can
+use the `--interactive` option. This flag allows you to confirm each stage
+before running.
+
+```dvc
+$ dvc exp run --interactive
+Going to reproduce stage: 'train'... continue? [y/n]
+```
+
+> Note that `dvc exp run` is an experimentation-specific alternative to
+> `dvc repro`.
+
+## (Hyper)parameters
+
+<abbr>Parameters</abbr> are the values that modify the underlying code's
+behavior, producing different experiment results. Machine learning
+experimentation, for example, involves searching hyperparameters that improve
+the resulting model metrics.
+
+In DVC projects, parameters should be read by the code from _parameter files_
+(`params.yaml` by default). DVC parses these files to track individual param
+values. When a tracked param is changed, `dvc exp run` invalidates any stages
+that depend on it, and reruns the experiment.
+
+> Parameters can be defined in `dvc.yaml` directly or through `dvc stage add`.  
+> 📖 See `dvc params` for more details.
+
+For a params file named `params.yaml` with the contents
+
+```yaml
+model:
+  learning_rate: 0.0001
+```
+
+You can specify the parameter dependency as
+
+```dvc
+$ dvc stage add -n train \
+                --parameter model.learning_rate \
+                --outs ...
+```
+
+> ⚠️ DVC does not check whether the parameters are actually used in your code.
+
+<details>
+
+#### Non-default parameter files
+
+DVC allows param files in YAML 1.2, JSON, TOML, and Python formats. When your
+parameters file is named something other than `params.yaml`, you need to specify
+it in both stage description and `dvc exp run`. For example using
+`myparams.toml`:
+
+```dvc
+$ dvc stage add -n train \
+                -p myparams.toml:learning_rate \
+                ...
+
+$ dvc exp run -S myparams.toml:learning_rate = 0.0001
+```
+
+</details>
+
+### Updating experiment parameters on-the-fly
+
+DVC allows to update the parameters from command line when running
+`dvc experiments`. The `--set-param` (`-S`) option takes a parameter name and
+its value, and updates the params file before the run.
+
+```dvc
+$ dvc exp run --set-param model.learning_rate=0.0002
+```
+
+> Note that parameters are attached to experiments so you can view them together
+> with `dvc exp show` and `dvc exp diff`.
+
+To set more than one param for the same experiment, use the `-S` option multiple
+times, or supply a comma-delimited list to `-S`:
+
+```dvc
+$ dvc exp run -S learning_rate=0.001 -S units=128
+# is equivalent to
+$ dvc exp run -S learning_rate=0.001,units=128
+```
+
+> ⚠️ Note that DVC doesn't check whether parameters given to `--set-param` are
```

Comment:
This is another case where we could prioritize this if it's important enough to be mentioned in the guide: https://github.com/iterative/dvc/issues/5477

Reply:
Well since it's not a bug and that issue has p3 I doubt it'll be implemented soon so I think it's OK to leave this note for now.